### ISSUE TITLE
[0.4.10] Invalid scripts?

### ISSUE THREAD

**Issue by hvdklauw** [NillerMedDild/Enigmatica5#301]
**Modpack version**
0.4.10

**Issue**
When loading the game it says invalid scripts and throws some errors.

**Logs**
```
[20:47:02.070][DONE][CLIENT][ERROR] ERROR normal\recipetypes\powah\energizing.zs:12:17: Could not cast IData[] to List<IData>?
[20:47:02.070][DONE][CLIENT][ERROR] ERROR functions\unification.zs:386:13: Could not cast IData[] to List<IData>?
```
[crafttweaker.log](https://github.com/NillerMedDild/Enigmatica5/files/4624055/crafttweaker.log)

**Comment by NielsPilgaard** [comment]
I'm working to resolve that issue, it happens when mods load in a specific order :/ there isn't actually an error in the scripts. Are you using MultiMC?

**Comment by hvdklauw** [comment]
Nope, GDlauncher

**Comment by NielsPilgaard** [comment]
Alright, is this the first time you've seen the issue? Restarting can fix it.

**Comment by hvdklauw** [comment]
Ok, hold on.. this will take a while :D

**Comment by hvdklauw** [comment]
confirmed, restarted and didn't show up.

Now I wonder if a simple /reload would have fixed it too

**Comment by NielsPilgaard** [comment]
It would not, the datapack is not the issue, the loading of the mods is - Glad it worked though. I'm working for a reliable fix, but it's gonna take a while.

**Comment by NielsPilgaard** [comment]
I've been told the bug has been fixed as of 0.4.12. If anyone encounters the issue again, please contact me asap :)
